src/utils/console.js

```diff
@@ -0,0 +1,20 @@
+import { Console } from "@woowacourse/mission-utils";
+import { MESSAGES, GAME_RULES } from "../constants/index.js";
+
+export const getCarNames = async () => {
+  const carName = await Console.readLineAsync(MESSAGES.CAR_NAME_INPUT);
+  return carName.split(GAME_RULES.COMMA).map((carName) => carName.trim());
+};
```

Comment:
이 부분도 마찬가지로 순수 함수를 유지하고 테스트 용이성 + 재사용성을 확보하려면 2개로 쪼개는게 좋을 것 같아요

```suggestion
export const readCarNamesInput = async () => {
  const carNameInput = await Console.readLineAsync(MESSAGES.CAR_NAME_INPUT);
  return carNameInput;
};

export const parseCarNames = (input) => 
  input.split(GAME_RULES.COMMA).map((carName) => carName.trim());
  ```

Reply:
말씀해주신 방법도 좋은 것 같습니다👍👍👍